azure-pipelines.yml

```diff
@@ -212,6 +212,11 @@ stages:
         inputs:
           path: 'TestResults'
           artifact: DotnetTestResultsNet6
+      
+      - task: PublishTestResults@2
+        inputs:
+          testRunner: VSTest
+          testResultsFiles: 'TestResults/**/*.trx'
```

Comment:
On the scanner side, we use this
```
sonar.cs.vstest.reportsPaths="$(Build.SourcesDirectory)/TestResults/*.trx"
```
can we use the same thing here with the `Build.SourcesDirectory` ?

Reply:
Only relative paths are used by the other tasks of that step, e.g.
```yaml
      - task: PublishPipelineArtifact@1
        displayName: 'Save test results files'
        inputs:
          path: 'TestResults'
          artifact: DotnetTestResultsNet6
```
So I would stick with that here.

The defaults settings of that tasks according to the docs are:
```yaml
    testResultsFiles: '**/TEST-*.xml' 
    searchFolder: '$(System.DefaultWorkingDirectory)' # Optional
```

So maybe like so:
```yaml
    testResultsFiles: '*.trx' 
    searchFolder: '$(System.DefaultWorkingDirectory)/TestResults'
```

I will play around with the settings to see what is working.

Reply:
I prefer the original simple solution if it works correctly. I was just asking about consistency here. We don't have to overcomplicate it